data/json/recipes/armor/storage.json

```diff
@@ -172,7 +193,7 @@
     "autolearn": true,
     "reversible": true,
     "using": [ [ "tailoring_leather_small", 2 ], [ "clasps", 1 ] ],
-    "proficiencies": [ { "proficiency": "prof_closures" }, { "proficiency": "prof_leatherworking" } ]
+    "proficiencies": [ { "proficiency": "prof_closures" }, { "proficiency": "prof_leatherworking_basic" } ]
```

Comment:
What decides as to whether just basic leatherworking or both is used?

Reply:
Generally if it's just adding a little strap to something or working with something already made, it's basic. A few of the really simple pouches as well. Almost all the time it should be the full proficiency